win/rl/winrl.cc

```diff
@@ -670,6 +721,7 @@ NetHackRL::destroy_nhwindow_method(winid wid)
 {
     DEBUG_API("rl_destroy_nhwindow(wid=" << wid << ")" << std::endl);
     windows_[wid].reset(nullptr);
+    windows_.resize(wid);
```

Comment:
This seems strange?

(1) It renders the line before needless.
(2) It assumes no index `i >= wid` is currently used?

Reply:
Yes for some reason I thought the windows were stored in a stack-style data structure (largely verified in practice) but maybe this is all wrong. Either way should probably go read more code.